Database/Corrections/Automatic/sodBaseItems.lua

```diff
@@ -2821,14 +2821,6 @@ function SeasonOfDiscovery:LoadBaseItems()
             [itemKeys.vendors] = nil,
             [itemKeys.startQuest] = nil,
         },
-        [210187] = {
-            [itemKeys.name] = "Venture Co. Work Order",
-            [itemKeys.npcDrops] = {3988,4070,3992,3993},
-            [itemKeys.objectDrops] = nil,
-            [itemKeys.itemDrops] = nil,
-            [itemKeys.vendors] = nil,
-            [itemKeys.startQuest] = nil,
-        },
```

Comment:
We want to keep the entry. Just never edit any of the `sodBase` files with the exception of the `sodBaseQuests.lua` file for when you add a new quest from the scraper.

When scraping the latest wowhead state for SoD, we replace the entire base files to mirror that state.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5339/commits/2d618139cf5b19c5a1926c230a55a5dd9dd0360c